src/test/java/com/direwolf20/buildinggadgets/util/RegionBlockView.java

```diff
@@ -0,0 +1,68 @@
+package com.direwolf20.buildinggadgets.util;
+
+import com.direwolf20.buildinggadgets.common.building.Region;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.IBlockAccess;
+import net.minecraft.world.WorldType;
+import net.minecraft.world.biome.Biome;
+
+import javax.annotation.Nullable;
+
+public class RegionBlockView implements IBlockAccess {
```

Comment:
Please add some description of what it does and what it is meant for (for example what kind of tests is this meant for)